weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/CallStatusCodeChip.tsx

```diff
@@ -0,0 +1,23 @@
+import {Chip} from '@mui/material';
```

Comment:
This has some overlap with the work I did this week in https://github.com/wandb/weave/pull/1072

Reply:
👍 Nice callout - let's use yours with the nice symbol!

Reply:
OK, why don't we land this, and then I'll update that PR and re-request review.